src/trans-word-level/instantiate_word_level.cpp

```diff
@@ -247,30 +264,36 @@ void wl_instantiatet::instantiate_rec(exprt &expr)
   else if(expr.id()==ID_sva_eventually ||
           expr.id()==ID_sva_s_eventually)
   {
-    assert(expr.operands().size()==1);
-    
-    // For Fp to hold, we need one of the following:
-    // (1) at least one state from the current point
-    //     in time satisfies 'p' OR
-    // (2) no state from now on is repeated
-    //     (which means 'p' can still hold later)
+    const auto &p = to_unary_expr(expr).op();
 
-    // save the current time frame, we'll change it
-    
-    exprt::operandst disjuncts;
+    // The following needs to be satisfied for a counterexample
+    // to Fp:
+    // (1) No state up to the current state satisfies 'p'.
+    // (2) The current state is equal to any earlier state.
```

Comment:
Nit pick: I'd use "some" instead of "any"

Reply:
done